python/simpler/worker.py

```
@@ -3786,6 +3986,279 @@ def copy_from(self, dst: int, src: int, size: int, worker_id: int = 0) -> None:
+        """Unmap + free every still-registered host buffer (called from close())."""
+        with self._registry_lock:
+            entries = list(self._host_buf_registry.values())
+            self._host_buf_registry.clear()
+            self._rebuild_host_buf_snapshot()
+        for entry in entries:
+            try:
+                if self._worker is not None and getattr(self, "_hierarchical_started", False):
+                    self._broadcast_host_unmap(entry.token)
+            except Exception:  # noqa: BLE001
+                # A failed unmap broadcast must not strand the parent shm.
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)